Source/WebKit/GPUProcess/GPUProcess.cpp

```diff
@@ -426,6 +426,16 @@ void GPUProcess::setUseSCContentSharingPicker(bool use)
 #endif
 }
 
+void GPUProcess::enableMicrophoneMuteStatusAPI()
+{
+#if PLATFORM(COCOA)
+    CoreAudioSharedUnit::unit().setMuteStatusChangedCallback([weakProcess = WeakPtr { *this }] (bool isMuting) {
+        if (RefPtr process = weakProcess.get())
+            process->protectedParentProcessConnection()->send(Messages::GPUProcessProxy::SetMicrophoneMuteStatusChanged(isMuting), 0);
```

Comment:
Here and elsewhere: **Set**MicrophoneMuteStatus**Changed** is a confusing name. I would drop either `Set` or `Changed` - `setMicrophoneMuteStatus` or `microphoneMuteStatusChanged`